modules/shared_data/src/main/scala/org/elpaca_metagraph/shared_data/app/ApplicationConfig.scala

```diff
@@ -50,6 +54,25 @@ object ApplicationConfig {
     apiUrl  : Option[String]
   )
 
+  case class WalletsInfo(
+    address      : Address,
+    minimumAmount: Long,
+    rewardAmount : Long,
+    afterDate    : LocalDate
```

Comment:
What is the date in that context? Which timezone, how it's calculated?

Reply:
This date is UTC date and it's provided in the configs so we can filter transactions sent/received greater than this date